ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/validate-synthea-load.py

```diff
@@ -0,0 +1,178 @@
+#
+# Script for validating via a quick smoke test that a synthea load successfully made it into the database by ensuring that 
+# values from the load files are in the database, and the correct number of benes were loaded. 
+#
+# Args:
+# 1: bene id start (inclusive, taken from previous end state properties / synthea properties file)
+# 2: bene id end (exclusive, taken from new output end state properties)
+# 3: db data string (db usernames and passwords for each environment DB, in this format (including quotes): "testUsername,testPassword,prodSbxUsername,prodSbxPassword,prodUsername,prodPassword")
+# 4: file system location of synthea folder
+# 5: which environments to check, should be a single comma separated string consisting of test,sbx,prod or any combo of the three (example "test,sbx,prod" or "test")
+#
+# Example runstring: python3 ./validate-synthea-load.py -10000004009994 -10000004010094 $DB_STRING ~/Git/synthea-2 "test,sbx"
+#
+# Requires psycopg2 installed
+#
+
+import sys
+import psycopg2
+import fileinput
+import shlex
+
+def validate_synthea_load(args):
+    """
+    Validates key data from the load files and number of benes expected were loaded into the database.
+    """
+    
+    bene_id_start = args[0]
+    bene_id_end = args[1]
+    db_data = args[2].split(',')
+    synthea_folder_filepath = args[3]
+    envs = args[4].split(',')
+    
+    if not synthea_folder_filepath.endswith('/'):
+        synthea_folder_filepath = synthea_folder_filepath + "/"
+    synthea_output_folder = synthea_folder_filepath + "output/bfd/"
+    
+    test_db_string = f"postgres://{db_data[0]}:{db_data[1]}@bfd-test-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
+    prod_sbx_db_string = f"postgres://{db_data[2]}:{db_data[3]}@bfd-prod-sbx-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
+    prod_string = f"postgres://{db_data[4]}:{db_data[5]}@bfd-prod-aurora-cluster.cluster-ro-clyryngdhnko.us-east-1.rds.amazonaws.com/fhirdb"
```

Comment:
potential refactoring here since we have this in two places now and involves hardcoded strings.

Reply:
Indeed, I have a ticket to replace the hardcoded strings and usernames/passwords with SSM instead, so this should get replaced in a followup PR.

Similar answer to @mjburling about the db creds, I'll be adjusting that soon in BFD-2097 which im working on now.